test/org/sagebionetworks/bridge/hibernate/HibernateHelperTest.java

```diff
@@ -41,14 +42,16 @@
 
     private HibernateHelper helper;
     private Session mockSession;
-
+    private SessionFactory mockSessionFactory;
+    
     @Before
     public void setup() {
         // mock session
         mockSession = mock(Session.class);
+        mockSessionFactory = mock(SessionFactory.class);
 
         // Spy Hibernate helper. This allows us to mock execute() and test it independently later.
-        helper = spy(new HibernateHelper());
+        helper = spy(new HibernateHelper(mockSessionFactory, new AccountPersistentExceptionConverter()));
```

Comment:
Would it be better to create a test converter (or a mock converter) instead of taking a hard dependency on AccountPersistentExceptionConverter?